recipes/gdist/meta.yaml

```diff
@@ -0,0 +1,47 @@
+package:
+  name: gdist
+  version: "1.0.3"
+
+source:
+  md5: 98547eac8e7c0b5a7a600e12ae5c9c4c
```

Comment:
`sha256` is [preferred](https://conda-forge.org/docs/meta.html#populating-the-hash-field). Looks like the value should be `466ae588a1277d5b085d8caff86721f7c1ede717431341afdb44e99c00926d46`.